### PR TITLE
merge: polish

### PR DESCRIPTION
**Pull request by tsibley**
See commit messages.

## Checklist

- [x] Automated checks pass
- [x] [Check][1] if you need to add a changelog message
- [x] [Check][2] if you need to add tests
- [x] [Check][3] if you need to update docs

[1]: https://github.com/nextstrain/augur/blob/-/docs/contribute/DEV_DOCS.md#updating-the-changelog
[2]: https://github.com/nextstrain/augur/blob/-/docs/contribute/DEV_DOCS.md#testing
[3]: https://github.com/nextstrain/augur/blob/-/docs/contribute/DEV_DOCS.md#when-to-update

### CODE REVIEW

**Comment by tsibley** [comment]
Rebased onto latest `master` so linkcheck passes.

**Comment by tsibley** [comment]
One more bit of error polishing coming in a few minutes…

File: augur/merge.py [hunk 1]
```diff
@@ -96,7 +104,7 @@ def run(args):
         raise AugurError(dedent(f"""\
             All metadata inputs must be assigned a name, e.g. with NAME=FILE.
 
-            The following inputs were missing a name:
+            The following {_n("input was", "inputs were", len(unnamed))} missing a name:
```
Comment (victorlin) [inline augur/merge.py:107]: Nice, I didn't know about `ngettext`. If/when this goes through I'd like to replace 37932e1d51604b21bfed39890757f4cde2ee490d.

File: augur/merge.py [hunk 2]
```diff
@@ -73,14 +81,14 @@ def register_parser(parent_subparsers):
...
-    input_group.add_argument("--metadata-delimiters", default=DEFAULT_DELIMITERS, nargs="+", action=ExtendOverwriteDefault, metavar="CHARACTER", help="delimiters to accept when reading a metadata file. Only one delimiter will be inferred.")
+    input_group.add_argument("--metadata-id-columns", default=DEFAULT_ID_COLUMNS, nargs="+", action=ExtendOverwriteDefault, metavar="COLUMN", help=f"Possible metadata column names containing identifiers, considered in the order given. Columns will be considered for all metadata tables. Only one ID column will be inferred for each table. (default: {' '.join(map(shquote_humanized, DEFAULT_ID_COLUMNS))})" + SKIP_AUTO_DEFAULT_IN_HELP)
+    input_group.add_argument("--metadata-delimiters", default=DEFAULT_DELIMITERS, nargs="+", action=ExtendOverwriteDefault, metavar="CHARACTER", help=f"Possible field delimiters to use for reading metadata tables, considered in the order given. Delimiters will be considered for all metadata tables. Only one delimiter will be inferred for each table. (default: {' '.join(map(shquote_humanized, DEFAULT_DELIMITERS))})" + SKIP_AUTO_DEFAULT_IN_HELP)
```
Comment (victorlin) [inline augur/merge.py:87]: +1 for these help text changes. Just noting that they're no longer consistent with [other instances](https://github.com/search?q=org%3Anextstrain+"names+of+possible+metadata+columns+containing+identifier+information"&type=code). We should consider aligning them better, maybe through a shared parser config or even just variables for help text.

(These have also propagated outside this repo to set_final_strain_name.py but I wouldn't worry about that as it's [slated for removal](https://github.com/nextstrain/private/issues/131))

**Review by victorlin — APPROVED** [review]
Looks good. Added a few non-blocking comments which would be good to address post-merge, maybe filed as new issues. I can write those up / just do it if it's easy enough.

File: augur/merge.py (hunk 1 above)
Comment (tsibley) [reply augur/merge.py:107]: Nod. I also think your solution in that commit is just fine. I started with something similar but it got uglier than I wanted, so I switched to `ngettext`.

File: augur/merge.py (hunk 2 above)
Comment (tsibley) [reply augur/merge.py:87]: Yeah, for sure. If you could open an issue for this that'd be great.

File: augur/merge.py (hunk 1 above)
Comment (victorlin) [reply augur/merge.py:107]: https://github.com/nextstrain/augur/pull/1596

File: augur/merge.py (hunk 2 above)
Comment (victorlin) [reply augur/merge.py:87]: https://github.com/nextstrain/augur/issues/1597

**Pull request merged**